remill/OS/OS.cpp

```diff
@@ -24,6 +25,19 @@ DEFINE_string(os, REMILL_OS, "Operating system name of the code being "
 
 namespace remill {
 
+OSName GetOSName(const llvm::Triple::OSType &os) {
```

Comment:
Lets make this take in the environment or file type here too, as a backup. So for example, if the file type is ELF, a.out, or COFF, default to linux, Mach-O for macOS, PE, etc. for Windows.

Reply:
Fixed in f12d641